phpunit/class-block-library-comment-template-test.php

```diff
@@ -13,8 +13,13 @@ class Block_Library_Comment_Template_Test extends WP_UnitTestCase {
 
 	private static $custom_post;
 	private static $comment_ids;
+	private static $per_page = 5;
 
 	public function setUp() {
+		parent::setUp();
+
+		update_option( 'page_comments', true );
```

Comment:
It looks like it's a common pattern to update options whenever necessary in the unit test:

https://github.com/WordPress/wordpress-develop/blob/7155e3c0832f1c9507ca0d50362d98fc7dcf868d/tests/phpunit/tests/comment/commentsTemplate.php#L31-L32